src/sparseml/pytorch/yolov8/val.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+from sparseml.pytorch.yolov8.trainers import SparseYOLO
+
+
+@click.command(
+    context_settings=(
+        dict(token_normalize_func=lambda x: x.replace("-", "_"), show_default=True)
+    )
+)
+@click.option(
+    "--task",
+    default="detect",
+    type=click.Choice(["detect", "segment", "classify"]),
+    help="Specify task to run.",
+)
+@click.option(
+    "--model",
+    default="yolov8n.yaml",
+    type=str,
+    help="i.e. yolov8n.pt, yolov8n.yaml. Path to model file",
+)
+@click.option(
+    "--data",
+    default="coco128.yaml",
+    type=str,
+    help="i.e. coco128.yaml. Path to data file",
+)
+@click.option(
+    "--save-json", default=False, is_flag=True, help="save results to JSON file"
+)
+@click.option(
+    "--save-hybrid",
+    default=False,
+    is_flag=True,
+    help="save hybrid version of labels (labels + additional predictions)",
+)
+@click.option(
+    "--conf",
+    default=None,
```

Comment:
let's set the default here (0.001 afair), to be more explicit

Reply:
Good catch, thanks!